docs/en/observability/cloud-monitoring/aws/monitor-aws-firewall-firehose.asciidoc

```diff
@@ -0,0 +1,120 @@
+== Step 2: Select a resource
+
+[role="screenshot"]
+image::firehose-networkfirewall-firewall.png[AWS Network Firewall]
+
+You can either use an existing AWS Network Firewall, or create a new one for testing purposes.
+
+Creating a Network Firewall is not trivial and is beyond the scope of this guide. For more information, see the AWS documentation on the https://docs.aws.amazon.com/network-firewall/latest/developerguide/getting-started.html[Getting started with AWS Network Firewall] guide.
+
+[discrete]
+[[firehose-firewall-step-three]]
+== Step 3: Create a stream in Amazon Data Firehose
+
+[role="screenshot"]
+image::firehose-networkfirewall-stream.png[Firehose stream]
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to Amazon Data Firehose.  
+
+. Click *Create Firehose stream* and choose the source and destination of your Firehose stream. Unless you are streaming data from Kinesis Data Streams, set source to `Direct PUT` and destination to `Elastic`. 
+
+. Provide a meaningful *Firehose stream name* that will allow you to identify this delivery stream later. 
+
+NOTE: For advanced use cases, source records can be transformed by invoking a custom Lambda function. When using Elastic integrations, this should not be required.
+
+Set the **Parameters** in the **Destination settings** section.
+
+[discrete]
+[[firehose-firewall-step-three-desination-settings-parameters]]
+=== Parameters
+
+Use the following parameters:
+
+[cols="1,1",options="header"]
+|===
+|Name
+|Value
+
+| `es_datastream_name`
+| `logs-aws.firewall_logs-default`
+
+|===
+
+The Firehose stream is ready to send logs to our Elastic Cloud deployment.
+
+[discrete]
+[[firehose-firewall-step-four]]
+== Step 4: Enable logging
+
+[role="screenshot"]
+image::firehose-networkfirewall-logging.png[AWS Network Firewall logging]
+
+The AWS Network Firewall logs have built-in logging support. It can send logs to Amazon S3, Amazon CloudWatch, and Amazon Kinesis Data Firehose.
+
+To enable logging to Amazon Data Firehose:
+
+. In the AWS console, navigate to the AWS Network Firewall service.
+
+. Select the firewall for which you want to enable logging.
+
+. In the *Logging* section, click *Edit*.
+
+. Select the *Send logs to* option and choose *Kinesis Data Firehose*.
+
+. Select the Firehose stream you created in the previous step.
+
+. Click *Save*.
+
+At this point, the Network Firewall will start sending logs to the Firehose stream.
+
+
+[discrete]
+[[firehose-firewall-step-five]]
+== Step 5: Visualize your Network Firewall logs in {kib}
+
+With the new logging settings in place, the Network Firewall starts sending log events to the Firehose stream.
+
+image::firehose-networkfirewall-data-stream.png[Firehose monitor Network Firewall logs]
+
+Navigate to {kib} and choose among the following monitoring options:
+
+- *Visualize your logs with Discover*
++
+image::firehose-networkfirewall-discover.png[Visualize Network Firewall logs with Discover]
```

Comment:
And we should add it here because this one is a screenshot

```suggestion
[role="screenshot"]
image::firehose-networkfirewall-discover.png[Visualize Network Firewall logs with Discover]
```